docs/hip-16.md

```diff
@@ -0,0 +1,96 @@
+---
+hip: 16
+title: Entity Auto Renewal
+author: Leemon Baird (@lbaird), Nosh Mody (@noshmody), Quan Nguyen (@qnswirlds)
+type: Standards Track
+category: Service
+status: Draft
+created: 2021-03-29
+discussions-to: <a URL pointing to the official discussion thread>
+updated:
+requires:
+replaces:
+superseded-by:
+---
+
+## Abstract
+
+When a Hedera entity is created, the payer account is charged enough hbars (as a rental fee) for the entity to stay active
+in the ledger state until consensus time passes its _expiration time_. Users can extend the expiration time of an entity by
+paying an extension fee via an update transaction. This HIP defines and discusses another mechanism to be implemented by
```

Comment:
Since not all entities have an update transaction.
```suggestion
paying an extension fee via an update transaction (if applicable). This HIP defines and discusses another mechanism to be implemented by
```